pkgs/data/fonts/nerdfonts/update.sh

```diff
@@ -20,7 +20,7 @@ while
   read -r name
   read -r url
 do
-    printf '  "%s" = "%s";\n' "${name%.*}" "$(nix-prefetch-url "$url")" >>"$dirname/shas.nix"
-done < <(jq -r '.assets[] | .name, .browser_download_url' <<<"$latest_release")
+    printf '  "%s" = "%s";\n' "${name%%.*}" "$(nix-prefetch-url "$url")" >>"$dirname/shas.nix"
+done < <(jq -r '.assets[] | .name, .browser_download_url | select(test("xz"))' <<<"$latest_release")
```

Comment:
I'm not much of a jq whiz so I did the simplest thing I could think of which was a regex match.

Reply:
Changed this around a bit to make the intention more obvious

`.assets[] | select(.name | test("xz")) | .name, .browser_download_url`